hermes/.cargo/config.toml

```diff
@@ -1,7 +1,7 @@
 # Use MOLD linker where possible, but ONLY in CI applicable targets.
 # cspell: words rustflags armv gnueabihf msvc nextest idents rustdocflags
 # cspell: words rustdoc lintfix lintrestrict testfast testdocs codegen testci testunit
-# cspell: words fmtchk fmtfix
+# cspell: words fmtchk fmtfix testcov
```

Comment:
Can we move these words to `project.dic` ?

Reply:
fixed